webpack.config.js

```diff
@@ -17,8 +17,8 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.js?$/, loaders: ['react-hot', 'babel?stage=1&optional[]=runtime'], exclude: /node_modules/ },
-            { test: /\.js$/, loader: 'babel?stage=1&optional[]=runtime', exclude: /node_modules\/(?!admin-config)/ },
+            { test: /\.js?$/, loaders: ['react-hot', 'babel?stage=1&optional[]=runtime&blacklist=useStrict'], exclude: /node_modules/ },
```

Comment:
Using [`with`](https://github.com/marmelab/react-admin/pull/28/files#diff-3314ac9023c48e7762430964caf957d8R7) forces me to disable `strict mode` on each files & add it manually...